assets/src/application/ui/layout/dropdownMenu/DropdownToggle.tsx

```diff
@@ -2,28 +2,31 @@ import React from 'react';
 import classNames from 'classnames';
 
 import { DropdownToggleProps } from './types';
-import { MenuToggle } from '@infraUI/layout/menu';
-import { EspressoIcon, Icon, withTooltip } from '@application/ui/display';
+import { EspressoIcon, Icon } from '@application/ui/display';
 import { IconButton } from '@chakra-ui/core';
+import { MenuToggle } from '@infraUI/layout/menu';
+import { Tooltip } from '@infraUI/display';
 
 const DropdownToggle = React.forwardRef<typeof MenuToggle, DropdownToggleProps>(
-	({ icon = Icon.MORE, isOpen, ...toggleProps }, ref) => {
+	({ icon = Icon.MORE, isOpen, tooltip, ...toggleProps }, ref) => {
 		const className = classNames('components-dropdown-menu__toggle', toggleProps.className, {
 			'is-opened': isOpen,
 		});
 
 		return (
-			<MenuToggle
-				as={IconButton}
-				// @ts-ignore
-				icon={() => <EspressoIcon icon={icon} />}
-				variant='ghost'
-				{...toggleProps}
-				className={className}
-				ref={ref}
-			/>
+			<Tooltip title={tooltip}>
+				<MenuToggle
+					as={IconButton}
+					// @ts-ignore
+					icon={() => <EspressoIcon icon={icon} />}
+					variant='ghost'
+					{...toggleProps}
+					className={className}
+					ref={ref}
+				/>
+			</Tooltip>
 		);
```

Comment:
Please add a `@todo` here to wrap it inside `withTooltip`

Reply:
added the comment